gpu-enabler/src/main/scala/com/ibm/gpuenabler/ColumnPartitionSchema.scala

```diff
@@ -36,7 +44,7 @@ private[gpuenabler] case object ColumnFormat extends PartitionFormat
 
 private[gpuenabler] object ColumnPartitionSchema {
 
-  // Since we are creating a runtime mirror using the class loader of current thread,
+  // Since we are creating a runtime mirror usign the class loader of current thread,
```

Comment:
nit: using?

Reply:
You are correct.
correct : using
error : usign